src/loggers/ConsoleLogger.ts

```diff
@@ -32,6 +34,17 @@ export class ConsoleLogger implements Logger<string> {
         return this.log(content, "Error");
     }
 
+    batchLog(content: BatchLogContent): Promise<void> {
+        if (this.loggerRepository) {
+            const logs = content.map(content =>
+                this.mapContentToLog(content.content, content.messageType)
+            );
+            return new BatchLogMessageUseCase(this.loggerRepository).execute(logs).toPromise();
```

Comment:
Typically we use all Promises or all Futures. I see that Logger uses Promises. Let's not change it, but just as a note.